### PR TITLE
Make jobs public

### PR DESCRIPTION
**Pull request by jonnyshaw89**
Branched from https://github.com/ONSdigital/eq-deploy/pull/35 so that should be merged 1st

Jobs should be visible without logging in to concourse.
You shouldn't be able to perform any actions without being logged in

### CODE REVIEW

**Review by dcdarrell9 — COMMENTED** [review]
This is fine, will review/approve after the other has been merged

**Comment by jonnyshaw89** [comment]
@dcdarrell9 👍

**Review by dcdarrell9 — APPROVED** [review]

**Review by dwyeradam — APPROVED** [review]

**Pull request merged**
